### DISCUSSION TITLE
Does this plugin work with Tomcat?

### DISCUSSION THREAD

**Discussion by aaabdallah** [StephenOTT/Camunda-Monitoring#24] [General]
Dear all,

Is this plugin compatible with running Camunda inside tomcat? I don't find the documentation covering that option ... just Camunda Run and Spring Boot installations. What about tomcat?

Many thanks in advance.

**Comment by MennaKhaled** [comment]
This is a blocker for me too, although I could get https://github.com/StephenOTT/camunda-prometheus-process-engine-plugin working.

Where is the equivalent of this folder in the new repo, where bpm-platform.xml is defined: https://github.com/StephenOTT/camunda-prometheus-process-engine-plugin/tree/master/docker/docker/camunda/conf
